src/CentreonModule/Tests/Infrastructure/Source/ModuleSourceTest.php

```diff
@@ -148,101 +177,107 @@ public function tearDown(): void
         $this->fs->unmount();
     }
 
-    public function testGetList()
+    public function testGetList(): void
     {
         $result = $this->source->getList();
 
         $this->assertTrue(is_array($result));
 
-        $result2 = $this->source->getList(static::$moduleNameMissing);
+        $result2 = $this->source->getList(self::$moduleNameMissing);
         $this->assertEquals([], $result2);
     }
 
-    public function testGetDetail()
+    public function testGetDetail(): void
     {
         (function () {
-            $result = $this->source->getDetail(static::$moduleNameMissing);
+            $result = $this->source->getDetail(self::$moduleNameMissing);
 
             $this->assertNull($result);
         })();
 
         (function () {
-            $result = $this->source->getDetail(static::$moduleName);
+            $result = $this->source->getDetail(self::$moduleName);
 
             $this->assertInstanceOf(Module::class, $result);
         })();
     }
 
-    public function testRemove()
+    /**
+     * @throws \Exception
+     * @return void
+     */
+    public function testRemove(): void
     {
         try {
-            $this->source->remove(static::$moduleNameMissing);
+            $this->source->remove(self::$moduleNameMissing);
         } catch (\Exception $ex) {
-            $this->assertEquals(static::$moduleNameMissing, $ex->getMessage());
+            $this->assertEquals(self::$moduleNameMissing, $ex->getMessage());
             $this->assertEquals(1, $ex->getCode()); // check moduleId
         }
 
-        $this->source->remove(static::$moduleName);
+        $this->source->remove(self::$moduleName);
     }
 
-    public function testUpdate()
+    /**
+    * @throws \Exception
+    * @return void
+    */
+    public function testUpdate(): void
     {
         try {
-            $this->assertNull($this->source->update(static::$moduleNameMissing));
+            $this->assertNull($this->source->update(self::$moduleNameMissing));
         } catch (\Exception $ex) {
-            $this->assertEquals(static::$moduleNameMissing, $ex->getMessage());
+            $this->assertEquals(self::$moduleNameMissing, $ex->getMessage());
             $this->assertEquals(1, $ex->getCode()); // check moduleId
         }
 
-        $this->source->update(static::$moduleName);
+        $this->source->update(self::$moduleName);
     }
 
-    public function testCreateEntityFromConfig()
+    /**
+    * @return void
+    */
+    public function testCreateEntityFromConfig(): void
     {
-        $configFile = static::getConfFilePath();
+        $configFile = self::getConfFilePath();
```

Comment:
Why are you changing the static keyword to self?

Reply:
It's a mistake I will fix in the next push cause I'm fixing some errors noted on jenkins

Reply:
reverted